interface/patient_file/front_payment.php

```diff
@@ -472,82 +472,81 @@ function toencounter(enc, datestr, topframe) {
-            text($patdata['lname']) . " (" . text($patdata['pubpid']) . ")" ?></td>
-        </tr>
-        <tr>
-            <td><?php echo xlt('Paid Via'); ?>:</td>
-            <td><?php echo generate_display_field(array('data_type' => '1', 'list_id' => 'payment_method'), $payrow['method']); ?></td>
-        </tr>
-        <tr>
-            <td><?php echo xlt('Check/Ref Number'); ?>:</td>
-            <td><?php echo text($payrow['source']) ?></td>
-        </tr>
-        <tr>
-            <td><?php echo xlt('Amount for This Visit'); ?>:</td>
-            <td><?php echo text(oeFormatMoney($payrow['amount1'])) ?></td>
-        </tr>
-        <tr>
-            <td>
+<body>
+    <div class="text-center">
+        <p><h2><?php echo xlt('Receipt for Payment'); ?></h2>
+
+        <p><?php echo text($frow['name']) ?>
+        <br /><?php echo text($frow['street']) ?>
+        <br /><?php echo text($frow['city'] . ', ' . $frow['state']) . ' ' .
+            text($frow['postal_code']) ?>
+        <br /><?php echo text($frow['phone']) ?>
+
+        <p>
+        <table class="table table-borderless">
+            <tr>
+                <td><?php echo xlt('Date'); ?>:</td>
+                <td><?php echo text(oeFormatSDFT(strtotime($payrow['dtime']))) ?></td>
+            </tr>
+            <tr>
+                <td><?php echo xlt('Patient'); ?>:</td>
+                <td><?php echo text($patdata['fname']) . " " . text($patdata['mname']) . " " .
+                text($patdata['lname']) . " (" . text($patdata['pubpid']) . ")" ?></td>
+            </tr>
+            <tr>
+                <td><?php echo xlt('Paid Via'); ?>:</td>
+                <td><?php echo generate_display_field(array('data_type' => '1', 'list_id' => 'payment_method'), $payrow['method']); ?></td>
+            </tr>
+            <tr>
+                <td><?php echo xlt('Check/Ref Number'); ?>:</td>
+                <td><?php echo text($payrow['source']) ?></td>
+            </tr>
+            <tr>
+                <td><?php echo xlt('Amount for This Visit'); ?>:</td>
+                <td><?php echo text(oeFormatMoney($payrow['amount1'])) ?></td>
+            </tr>
+            <tr>
+                <td>
+                <?php
+                if ($_REQUEST['radio_type_of_payment'] == 'pre_payment') {
+                    echo xlt('Pre-payment Amount');
+                } else {
+                    echo xlt('Amount for Past Balance');
+                }
+                ?>
+                :</td>
+                <td><?php echo text(oeFormatMoney($payrow['amount2'])) ?></td>
+            </tr>
+            <tr>
+                <td><?php echo xlt('Received By'); ?>:</td>
+                <td><?php echo text($payrow['user']) ?></td>
+            </tr>
+        </table>
+
+        <div id='hideonprint'>
+            <p>
+            <input type='button' value='<?php echo xla('Print'); ?>' id='printbutton' />
+
```

Comment:
Hi @bradymiller  and @tywrenn 

Can you guide me how to go to the Receipt for Payment page? In here, I only remove the body background color and <center> tag, but I think it still needs to refine, so I would like to see the page.

Reply:
> Can you guide me how to go to the **Receipt for Payment page** ? 
Hi @stu01509 , once you click on `Generate Invoice` it will bring you to the Receipt.